resources/splunk_installer.rb

```diff
@@ -0,0 +1,92 @@
+#
+# Author: Dang H. Nguyen <dang.nguyen@disney.com>
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+require_relative '../libraries/helpers'
+
+resource_name :splunk_installer
+default_action :run
```

Comment:
You can skip this in a custom resource

Reply:
skip which line or all of these lines being changed here?

Reply:
Looks like you nuked the wrong line here

Reply:
6e275f7b77cb09b85fe507b024253e67b73e23ee added back the line